ci-operator/step-registry/single-node/conf/single-node-conf-commands.sh

```diff
@@ -21,6 +26,10 @@ if not "controlPlane" in cfg:
 
 cfg["controlPlane"]["replicas"] = 1
 
+# Single Node Openshift requires extra memory and compute resources
+assert "aws" in cfg["controlPlane"]["platform"], "Only AWS single-node is supported for now"
```

Comment:
@omertuc dont we support gcp as well?

Reply:
Never tried it personally. For now I've hardcoded the aws instance type so gcp would break unless we add gcp instance types. We can add them whenever we want to run jobs on gcp, but what's the point?